src/Arcus.WebApi.Unit/Correlation/TelemetryCorrelationTests.cs

```diff
@@ -0,0 +1,114 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Arcus.WebApi.Correlation;
+using Arcus.WebApi.Unit.Hosting;
+using Newtonsoft.Json;
+using Serilog.Events;
+using Xunit;
+using static Arcus.WebApi.Unit.Correlation.CorrelationController;
+
+namespace Arcus.WebApi.Unit.Correlation
```

Comment:
I'd suggest one more test - What if you do 2 sequential calls, do they get the right correlation info?

Reply:
How you mean? Like two independent calls with both diff operation/transaction ID's?

Reply:
2 independent GETs rather than just one to verify if the correlation info is set correctly for both calls

Reply:
Isn't this test: `SendRequest_WithSerilogCorrelationEnrichment_ReturnsOkWithDifferentOperationIdAndDifferentTransactionId` doing just that?

Reply:
Yes, I've missed that one sorry.